apps/console/src/auth.jsp

```diff
@@ -0,0 +1,19 @@
+<%
```

Comment:
Shall we add the license header?

Reply:
Adding the license header results in the whole page being sent as a response instead of the JSON object. So, the license has to be omitted until a fix is found.

Reply:
Resolved with 21cbba4b